api/app/logs/__init__.py

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: MPL-2.0
```

Comment:
Do we need a logs app?

Reply:
Why not? aint it nice to have all logs configurations in a single place instead of scattered in utils files / settings files

Reply:
cause you point out what code would be unnecessary?

Reply:
Not sure what you want me to do here. I was just asking for some clarification. Do not get me wrong I have nothing against this approach 😄

Reply:
You are asking if all the code is really needed, so i was asking if you could point out the code that was not needed.

Yes most of the code can be removed. 
But the reason I added an extra filter is to have more control over the logging of the django queries to help debugging the queries

And the seperate Handler I added is to have a nicer and more easier viewing of the logs that are printed to the console during development. 

But yes both can be deleted but they are kinda nice to have for development reason in my opinion